kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2761,13 +2761,16 @@ object KyuubiConf {
       .version("1.10.0")
       .fallbackConf(OPERATION_INCREMENTAL_COLLECT)
 
-  val ENGINE_SPARK_OPERATION_INCREMENTAL_COLLECT_SKIP_CANCEL_GROUP: ConfigEntry[Boolean] =
-    buildConf("kyuubi.engine.spark.operation.incremental.collect.skip.cancel.group")
-      .doc("When true, the job group cancellation in executeStatement method " +
-        "will be skipped on incremental collect mode")
+  val ENGINE_SPARK_OPERATION_INCREMENTAL_COLLECT_CANCEL_JOB_GROUP: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.spark.operation.incremental.collect." +
+      "cancelJobGroupAfterExecutionFinished")
```

Comment:
let's keep it in one line, for searching. you can disable the style check in this case